modules/virt-about-kubemacpool.adoc

```diff
@@ -15,6 +15,5 @@ Virtual machine instances created from that virtual machine retain the assigned
 KubeMacPool does not handle virtual machine instances created independently from a virtual machine.
 ====
 
-KubeMacPool is disabled by default.
-Enable a MAC address pool for a namespace by applying the KubeMacPool label to the namespace.
-
+KubeMacPool is enabled by default when you install {VirtProductName}.
+You can disable a MAC address pool for a namespace by adding the KubeMacPool `ignore` label to the namespace. Re-enable KubeMacPool for the namespace by removing the label.
```

Comment:
I think it may be better to call it "disabling label". but I'm not totally sure about that. what do you think?

```suggestion
You can disable a MAC address pool for a namespace by adding the KubeMacPool disabling label to the namespace. Re-enable KubeMacPool for the namespace by removing the label.
```

Reply:
@dankenigsberg what do you think?

Reply:
I don't know... I prefer to be explicit as `ignore` is. However, I also prefer to be precise, and the precise label is `mutatevirtualmachines.kubemacpool.io=ignore`, not just `ignore`.

Reply:
then I think it should look like this:
```suggestion
You can disable a MAC address pool for a namespace by adding the KubeMacPool `mutatevirtualmachines.kubemacpool.io=ignore` label to the namespace. Re-enable KubeMacPool for the namespace by removing the label.
```

Reply:
maybe even a shorter version
```suggestion
You can disable a MAC address pool for a namespace by adding the `mutatevirtualmachines.kubemacpool.io=ignore` label to the namespace. Re-enable KubeMacPool for the namespace by removing the label.
```

Reply:
Updated as suggested. Thanks for the input, folks

Reply:
can you update the preview link?

Reply:
Just pushed the commit. The preview link should update in ~5 minutes